containerized_ml_models/api/api.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from xgboost import XGBModel
+from flask import Flask, jsonify, request
+from containerized_ml_models.api import io
+
+MODEL_PATH: str = "containerized_ml_models/api/model.pkl"
+model: XGBModel = io.load_model(MODEL_PATH)
+
+app: Flask = Flask(__name__)
+
+
+@app.route("/ping", methods=["GET"])
+def ping():
+    return jsonify({"ping": "success"})
+
+
+@app.route("/predict", methods=["POST"])
+def predict():
+    content: list[list[float]] = request.json
+    input_data: np.ndarray = np.array(content)
+    prediction = model.predict(input_data).tolist()
```

Comment:
Dobrą praktyką jest spójne podejście. W tym przypadku jeżeli już chcesz typować to raczej powinieneś wszystkie zmienne bo dlaczego jedne tak a inne nie?